pkg/synopsysctl/cmd_update.go

```diff
@@ -1176,11 +1183,94 @@ func addUpdateOperatorFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", namespace, "Namespace of the Synopsys Operator instance")
 }
 
+func updatePolaris(polarisObj polaris.Polaris, flagset *pflag.FlagSet) (*polaris.Polaris, error) {
+	if err := updatePolarisCobraHelper.SetCRSpec(polarisObj); err != nil {
+		return nil, err
+	}
+
+	spec, err := updatePolarisCobraHelper.GenerateCRSpecFromFlags(flagset)
+	if err != nil {
+		return nil, err
+	}
+	newSpec := spec.(polaris.Polaris)
+
+	return &newSpec, nil
+}
+
+// updatePolarisCmd updates a Polaris instance
+var updatePolarisCmd = &cobra.Command{
+	Use:           "polaris NAME",
+	Example:       "synopsyctl update polaris <name>",
+	Short:         "Update a Polaris instance",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 0 {
```

Comment:
The "Use" above implies the command takes in one argument.

Reply:
fixed